packages/web-components/src/components/ic-side-navigation/ic-side-navigation.tsx

```diff
@@ -42,41 +42,40 @@ import { IcTopBar } from "./ic-side-navigation.types";
 export class SideNavigation {
   @Element() el: HTMLIcSideNavigationElement;
   /**
-   * Title to display.
+   * The title of the app to be displayed.
    */
   @Prop() appTitle!: string;
   /**
-   * Status of application
+   * The status of the application to be displayed.
```

Comment:
I think we should use 'app' instead of 'application' - this goes with [what we've said on our guidance site](https://design.sis.gov.uk/styles/content-style/voice-and-tone#swap-formal-words-for-normal-ones:~:text=Approximate-,App,Application,-Ask)